clevis/Sanity/clevis-luks-askpass-enabled/main.fmf

```diff
@@ -12,8 +13,10 @@ link:
   - relates: https://bugzilla.redhat.com/show_bug.cgi?id=2107081
 adjust:
   - enabled: false
-    when: distro < rhel-8.7
+    when: distro < rhel-8.6
     continue: false
   - enabled: false
-    when: distro < rhel-9.1
+    when: distro ~< rhel-9.0
```

Comment:
@lukaszachy Hi Lukas, can you please explain us more about this? @sopos said that this part of `adjust` is not necessary. Thanks!

Reply:
Not sure what "this' means for this :) so let me explain how it works and it is up to you to decide. Documentation for context matching is https://fmf.readthedocs.io/en/stable/context.html

For 'major-version' there is a catch (explained in the table in the docs) that if _right_ side doesn't define minor version e the outcome is boolean instead of CannotDecide.
```python
>>> fmf.Context(distro="centos-8.7").matches("distro ~< centos-9")
True

>>> fmf.Context(distro="centos-8.7").matches("distro ~< centos-9.0")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/site-packages/fmf/context.py", line 571, in matches
    raise CannotDecide()  # It's up to callee how to treat this
fmf.context.CannotDecide
```

You need '~<' otherwise `distro < rhel-9` is true for every older rhel.

Reply:
the usage of the `~<` of course fixes the issue with excluding the test of rhel<9 but still the condition `distro ~< rhel-X.0` is generally redundant, so that part may be removed completely